src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -126,6 +133,16 @@ def strict_deps(self):
     return self.payload.strict_deps
 
   @property
+  def exports(self):
+    """A list of exported libraries, which will be accessible to dependents.
+
+    :return: See constructor.
+    :rtype: list
+    """
+    exports = [Address.parse(spec, relative_to=self.address.spec_path) for spec in self.payload.exports]
+    return [self._build_graph.get_target(addr) for addr in exports]
```

Comment:
On the one hand it would be nice to avoid the redundancy. OTOH, we want people to use `exports` sparingly, I assume, and if all exports were effectively dependencies it might cause people to use it as a synonym for `dependencies` without thinking too hard about what actually needs exporting.

Still, I'm leaning towards agreeing with Stu's comment, just providing the counterpoint.